### ISSUE TITLE
Crash on opening pending invoice details

### ISSUE THREAD

**Issue by rkfg** [btcontract/wallet#112]
There are several pending invoices that I have, they might have already expired (I might have created them yesterday, don't remember the exact date and it's not displayed anywhere). When I open any of them the app consistently crashes with the following log:
```
java.lang.IllegalArgumentException: Software rendering doesn't support hardware bitmaps
	at android.graphics.BaseCanvas.onHwBitmapInSwMode(BaseCanvas.java:632)
	at android.graphics.BaseCanvas.throwIfHwBitmapInSwMode(BaseCanvas.java:639)
	at android.graphics.BaseCanvas.throwIfCannotDraw(BaseCanvas.java:73)
	at android.graphics.BaseCanvas.drawBitmap(BaseCanvas.java:131)
	at android.graphics.Canvas.drawBitmap(Canvas.java:1588)
	at android.graphics.drawable.BitmapDrawable.draw(BitmapDrawable.java:548)
	at android.widget.ImageView.onDraw(ImageView.java:1429)
	at android.view.View.draw(View.java:21482)
	at android.view.View.draw(View.java:21356)
	at android.view.ViewGroup.drawChild(ViewGroup.java:4391)
	at android.view.ViewGroup.dispatchDraw(ViewGroup.java:4152)
	at android.view.ViewOverlay$OverlayViewGroup.dispatchDraw(ViewOverlay.java:254)
	at android.view.View.draw(View.java:21491)
	at android.widget.AbsListView.draw(AbsListView.java:4414)
	at android.view.View.draw(View.java:21356)
	at android.view.ViewGroup.drawChild(ViewGroup.java:4391)
	at android.view.ViewGroup.dispatchDraw(ViewGroup.java:4152)
	at android.view.View.draw(View.java:21354)
	at android.view.ViewGroup.drawChild(ViewGroup.java:4391)
	at android.view.ViewGroup.dispatchDraw(ViewGroup.java:4152)
	at android.view.View.draw(View.java:21354)
	at android.view.ViewGroup.drawChild(ViewGroup.java:4391)
	at android.view.ViewGroup.dispatchDraw(ViewGroup.java:4152)
	at android.view.View.draw(View.java:21354)
	at android.view.ViewGroup.drawChild(ViewGroup.java:4391)
	at android.view.ViewGroup.dispatchDraw(ViewGroup.java:4152)
	at android.view.View.draw(View.java:21354)
	at android.view.ViewGroup.drawChild(ViewGroup.java:4391)
	at android.view.ViewGroup.dispatchDraw(ViewGroup.java:4152)
	at android.view.View.draw(View.java:21354)
	at android.view.ViewGroup.drawChild(ViewGroup.java:4391)
	at android.view.ViewGroup.dispatchDraw(ViewGroup.java:4152)
	at android.view.View.draw(View.java:21485)
	at com.android.internal.policy.DecorView.draw(DecorView.java:826)
	at com.github.mmin18.widget.RealtimeBlurView$1.onPreDraw(RealtimeBlurView.java:225)
	at android.view.ViewTreeObserver.dispatchOnPreDraw(ViewTreeObserver.java:1088)
	at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:2830)
	at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:1800)
	at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:7751)
	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:967)
	at android.view.Choreographer.doCallbacks(Choreographer.java:791)
	at android.view.Choreographer.doFrame(Choreographer.java:726)
	at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:952)
	at android.os.Handler.handleCallback(Handler.java:883)
	at android.os.Handler.dispatchMessage(Handler.java:100)
	at android.os.Looper.loop(Looper.java:241)
	at android.app.ActivityThread.main(ActivityThread.java:7604)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:941)
```
It doesn't happen with the fresh invoices I created today, they open normally.

**Comment by btcontract** [comment]
Thanks, it's a known bug at this point, so be fixed in next release.

**Comment by btcontract** [comment]
Fixed in release `2.4.19`.
